frontend/components/commons/results/RecordExtraActions.vue

```diff
@@ -126,6 +113,16 @@ export default {
     close() {
       this.open = false;
     },
+    copyToClipboard() {
+      if (isObject(this.record.inputs)) {
+        const text = Object.keys(this.record.inputs).map((key) => {
+          return `${key}\r\n${this.record.inputs[key]}`;
+        });
```

Comment:
For text classification, the copied text does not keep the same layout that is shown in the record:

<img width="1537" alt="Screenshot 2022-07-21 at 11 29 00" src="https://user-images.githubusercontent.com/2518789/180180808-bf210960-019a-425e-bba1-bfe87f5a8221.png">

In the example below, the field names should be UPPERCASED (in my opinion) and every field should include a break line (maybe two) before the previous field (see the `headline` field in the image)

Also, I think we could skip the field name for cases when just one field (text) is found in the record. But it should be discussed

cc: @Amelie-V @dvsrepo @leiyre ???

Reply:
I agree with skip the name from the field when there is only one